.circleci/config.yml

```diff
@@ -364,7 +365,7 @@ workflows:
           requires:
             - deploy_release
       - performance-automation:
-          <<: *on-integ-and-version-tags
+          <<: *not-on-integ-branch
```

Comment:
shouldn't that be `on-integ-and-version-tags`?

Reply:
you're absolutly right @rafie . addressing it

Reply:
@rafie fixed